src/pages/index.js

```diff
@@ -59,16 +90,67 @@ function openImageModel(event) {
   imagePopup.open(event);
 }
 
-cardSection.renderItems(initialCards);
+function openEditProfilePicture() {
+  profilePicturePopup.open();
+}
+
+function openConfirmDelete(locationCardId, locationCard) {
+  const deletePopup = new PopupDelete(
+    ".confirm-delete-popup",
+    locationCardId,
+    locationCard,
+    deleteLocationCard
+  );
+  deletePopup.setEventListeners();
+  deletePopup.open();
+}
+
+function deleteLocationCard(locationCardId, locationCard) {
+  api.deleteCard(locationCardId).then(locationCard.remove());
```

Comment:
`.then(locationCard.remove());`

In `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well.